lib/config/config.go

```diff
@@ -307,12 +308,13 @@ func convertV13V14(cfg *Configuration) {
 
 	var newAddrs []string
 	for _, addr := range cfg.Options.GlobalAnnServers {
-		if addr != "default" {
-			uri, err := url.Parse(addr)
-			if err != nil {
-				panic(err)
-			}
-			uri.Path += "v2/"
+		uri, err := url.Parse(addr)
```

Comment:
"default" and so on are valid URLs with an empty scheme and will slip through this untouched.